inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vpu/frontend/frontend.hpp>
+
+#include <vector>
+#include <memory>
+#include <set>
+
+#include <vpu/stages/post_op_stage.hpp>
```

Comment:
Nitpick: better to place it before C++ libraries accroding to the code style.

Reply:
Fixed